index.bs

```diff
@@ -2293,35 +2293,70 @@ The W3C Council</h4>
 	the [=Chair of the Council=] <em class="rfc2119">should</em> dissolve the [=Council=]
 	and call for a new one to be convened.
 
-Issue: The following text about recusals is does not relfect the latest thinking.
-However, the latest thinking isn't fully stabilized yet,
-and thus not ready to replace this.
-See <a href="https://github.com/w3c/w3process/issues/278">https://github.com/w3c/w3process/issues/278</a>
-and in particular <a href="https://github.com/w3c/w3process/issues/278#issuecomment-995868961">https://github.com/w3c/w3process/issues/278#issuecomment-995868961</a>,
-as well as <a href="https://docs.google.com/document/d/1lB8Q_pvqxfN204FQHZ8D2HObxfo36qnguif9mGpEJTU/edit#">https://github.com/w3c/w3process/issues/278#issuecomment-995868961</a>.
-
-	No one shall be both a party and an adjudicator.
-	The person making either the decision being objected to or appealed,
-	and the person making the objection or appeal,
-	<em class=rfc2119>must</em> [=recuse=] themselves.
-	Participants <em rfc=2119>should</em> [=recuse=]
-	if they believe their participation would compromise the integrity of the Council decision,
-	and <em class=rfc2119>may</em> do so at any time.
-	However, recusal is disqualification from participation,
+	Individual recusal is not used in the Council. 
+	The potential ouncil themselves decide, uniformly, which reasons for each member to serve 
+	rise to a sufficient level for that potential member to be [=dismissed=].
+	No-one is automatically dismissed.
+				
+	The team assignee <em class=rfc2119>must</em> ensure a list is developed of all potential council members, 
+	and mark against each one possible reasons that they might need to be dismissed. 
+	The W3C community,
+	including members and team, and potential council members, 
+	<em class=rfc2119>must</em> be given an opportunity to contribute possible reasons to this list.
+
+	Example reasons (none of which are diagnostic) could include:
+	<ol>
+		<li>
+			They filed the formal objection, helped author it, 
+			or work for the same organization as the author/filer
+		<li>
+			They made the decision being objected to, were editor, or a major contributor
+		<li>
+			Their business depends on the question (in either direction)
+		<li>
+			They hold sufficiently strong pre-existing opinions in either direction 
+			(e.g. a well-known proponent or objector) that their ability 
+			to be open-minded may reasonably be in doubt
+		<li>	
+			They themselves offered the reason and asked that it be stated.
+		<li>
+			A member of the W3C membership or team feels that they 
+			should be dismissed and has presented a reason
+	</ol>
+
+	Before the Council forms, the team assignee presents the entire list of potenmtial members and collected reasons 
+	to the potential council; 
```

Comment:
between "presents the list[…]" and "the potential council then votes[…]", I think we should make it explicit that council members get a chance to explain / rebuke claims made about themselves. This the list is non filtered, it is not guaranteed to be free of bias or spin, or even to be factual, and thus an opportunity react is appropriate.